packages/compass-connections-navigation/src/item-actions.ts

```diff
@@ -50,23 +57,25 @@ export const notConnectedConnectionItemActions = ({
 };
 
 export const connectedConnectionItemActions = ({
+  connectionInfo,
   isReadOnly,
-  isFavorite,
   isPerformanceTabSupported,
 }: {
+  connectionInfo: ConnectionInfo;
   isReadOnly: boolean;
-  isFavorite: boolean;
   isPerformanceTabSupported: boolean;
 }): NavigationItemActions => {
-  const actions: NavigationItemActions = [];
-  if (!isReadOnly) {
-    actions.push({
+  const connectionManagementActions = notConnectedConnectionItemActions({
+    connectionInfo,
+    isEditDisabled: true,
+    editDisabledDescription: 'Cannot edit an active connection',
```

Comment:
I'm wondering if it really is so bad if we just remove the option from the list rather.